service/raft/raft_group0.cc

```diff
@@ -59,6 +59,17 @@ future<> raft_group0::uninit_rpc_verbs() {
     ).discard_result();
 }
 
+future<std::pair<raft::server_id, raft::server_info>> raft_group0::load_my_addr() {
```

Comment:
If you add load_my_addr(), load_or_create_my_addr should use it. load_or_create was added to reuse code.

Reply:
load_or_create_my_addr is not able to use load_my_addr, because load_my_addr calls on_internal_error in situations where we want to use load_or_create_my_addr :(